### PR TITLE
16801 Search Gov Maintenance Window - Show Banner

### PR DESCRIPTION
**Pull request by ariperez**
## Summary

Currently Search.gov has a maintenance window of Tue/Thu, 3-6pm EST.  If no results are returned during that time, we would like to show a banner as no results is most likely due to maintenance occurring at that time

## Related issue(s)

https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16801

## Testing done

- Tests that when zero results the banner doesn't show
- Tests that with zero results and within window, banner does show

## Screenshots

image

## What areas of the site does it impact?

Search 

## Acceptance criteria
- [ ] Banner shows during matininace windows on Tue/Thu 3-6pm EST when no results are returned, otherwise it does not show

### CODE REVIEW

File: src/applications/search/containers/SearchApp.jsx [hunk 1]
```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { withRouter } from 'react-router';
 import { connect } from 'react-redux';
+import moment from 'moment-timezone';
```
Comment (randimays) [inline src/applications/search/containers/SearchApp.jsx:5]: **Consider**: We should be moving away from using `moment` and `moment-timezone` as it is a much larger package/dependency than others like `date-fns`. If it's not a simple refactor to use another library, don't worry about it; just a "consider" here.

**Review by randimays — COMMENTED** [review]
I looked through the tickets / PRs related to this. I think the time-based handling we're doing here is the best solution 👍🏻 Thanks for your efforts on this!

File: src/applications/search/containers/SearchApp.jsx (hunk 1 above)
Comment (ariperez) [reply src/applications/search/containers/SearchApp.jsx:5]: good feedback, I'll try to use date-fns

**Review by randimays — APPROVED** [review]

**Review by chriskim2311 — APPROVED** [review]
LGTM!

**Pull request merged**